conf/cluster.config

```diff
@@ -42,20 +42,20 @@
 //}
 
 //uncomment for SLURM cluster
-//process {
-//  withLabel: longtime {
-//    executor='slurm'
-//    //queue='long'
-//    clusterOptions = '-t 3-00:00:0 --mem=6G'
-//  }
-//  withLabel: shorttime {
-//    executor='slurm'
-//    //queue='short'
-//    clusterOptions = '-t 00-01:00:0 --mem=6G'
-//  }
-//  withLabel: highmem {
-//    executor='slurm'
-//    //queue='short'
-//    clusterOptions = '-t 00-20:00:0 --mem=80G'
-//  }
-//}
+process {
+  withLabel: longtime {
+    executor='slurm'
+    //queue='default'
+    clusterOptions = '-t 3-00:00:0 --mem=6G'
+  }
+  withLabel: shorttime {
+    executor='slurm'
+    //queue='default'
+    clusterOptions = '-t 00-01:00:0 --mem=6G'
+  }
+  withLabel: highmem {
+    executor='slurm'
+    queue='bigmem'
+    clusterOptions = '-t 00-20:00:0 --mem=80G'
+  }
+}
```

Comment:
can you revert this changes?
I think it is better to to leave the default without any cluster environment.
It is also documented like this in the docs